.github/workflows/nightly.yml

```diff
@@ -0,0 +1,80 @@
+name: Publish Nightly Builds
+
+on:
+  push:
+    branches:
+      - master
+env:
+  REPOSITORY: async-profiler/async-profiler
+  RELEASE_ID: 175290112
+jobs:
+  delete-all-but-last-five-commits:
```

Comment:
Why do we need to delete these?

Reply:
Andrei wants to keep only the latest 5 commits

Reply:
Yeah, we don't need to store all artifacts indefinitely. At the same time, it should be clear for a user which artifact is the latest. Currently, it's not obvious at all: there is a bunch of similar files which differ only by a commit hash.
Suggestions are welcome. One idea is to update release notes with the links to the most recent artifacts.

Reply:
We could add the day in `yyyy-MM-dd` format to the archive filename, latest should have the word latest in it:

* async-profiler-3.0-2024-10-15-xxxxxxxxx-linux-x64.tar.gz
* async-profiler-3.0-2024-10-16-xxxxxxxxx-linux-x64.tar.gz
* async-profiler-3.0-2024-10-17-xxxxxxxxx-linux-x64.tar.gz
* async-profiler-3.0-2024-10-18-xxxxxxxxx-linux-x64.tar.gz
* async-profiler-3.0-2024-10-19-xxxxxxxxx-linux-x64.tar.gz
* async-profiler-3.0-latest-linux-x64.tar.gz - same as async-profiler-3.0-2024-10-19-xxxxxxxxx-linux-x64.tar.gz

This way, the link to latest will not change by the hash or a date, and it will be clear which day commits belong to, without checking from git history.

Reply:
currently it runs on every commit, so day is not specific enough. But duplicating the latest one and naming it latest makes sense, thanks!